packages/datetime/test/common/dateUtilsTests.tsx

```diff
@@ -39,4 +39,75 @@ describe("dateUtils", () => {
+
+        const time = DateUtils.getDateOnlyWithTime(date);
+
+        assert.equal(time.getFullYear(), 1899);
+        assert.equal(time.getMonth(), 11);
+        assert.equal(time.getDay(), 0);
+
+        assert.equal(time.getHours(), 14);
+        assert.equal(time.getMinutes(), 10);
+        assert.equal(time.getSeconds(), 10);
+        assert.equal(time.getMilliseconds(), 600);
+    });
+
+    describe("isTimeGreaterThan", () => {
+        it("returns true if given time is greater than another time", () => {
+            const date = new Date(1995, 6, 30, 14, 22, 30, 600);
+            const date2 = new Date(1995, 6, 30, 14, 20, 30, 600);
+
+            assert.isTrue(DateUtils.isTimeGreaterThan(date, date2));
+        });
+
+        it("returns false if given time is smaller than another time", () => {
+            const date = new Date(1995, 6, 30, 14, 10, 50, 900);
+            const date2 = new Date(1995, 6, 30, 14, 20, 30, 600);
+
+            assert.isFalse(DateUtils.isTimeGreaterThan(date, date2));
+        });
+    });
+
+    describe("isTimeInRange", () => {
+        // Note that year, month, and day are always ignored
+        const minTime = new Date(1995, 6, 30, 14, 20, 30, 600);
+        const maxTime = new Date(1995, 6, 30, 18, 40, 10, 200);
+
+        it("returns true if given time is in range", () => {
+            const time = new Date(1995, 6, 30, 17, 0, 0, 0);
+            assert.isTrue(DateUtils.isTimeInRange(time, minTime, maxTime));
+        });
+
+        it("returns true if given time is in range, and minTime > maxTime", () => {
+            const minTimeBeforeMidnight = new Date(1995, 6, 30, 22, 0, 0, 0);
+            const maxTimeAfterMidnight = new Date(1995, 6, 30, 2, 0, 0, 0);
+
+            const timeAfterMidnight = new Date(1995, 6, 30, 1, 0, 0, 0);
+            assert.isTrue(DateUtils.isTimeInRange(timeAfterMidnight, minTimeBeforeMidnight, maxTimeAfterMidnight));
+
+            const timeBeforeMidnight = new Date(1995, 6, 30, 23, 0, 0, 0);
+            assert.isTrue(DateUtils.isTimeInRange(timeBeforeMidnight, minTimeBeforeMidnight, maxTimeAfterMidnight));
+        });
+
+        it("returns false if given time is not in range, and minTime > maxTime", () => {
+            const minTimeBeforeMidnight = new Date(1995, 6, 30, 22, 0, 0, 0);
+            const maxTimeAfterMidnight = new Date(1995, 6, 30, 2, 0, 0, 0);
+
+            const time = new Date(1995, 6, 30, 16, 0, 0, 0);
+            assert.isFalse(DateUtils.isTimeInRange(time, minTimeBeforeMidnight, maxTimeAfterMidnight));
+        });
+
+        it("returns false if given time is smaller than minTime", () => {
+            const time = new Date(1995, 6, 30, 13, 10, 50, 900);
+            assert.isFalse(DateUtils.isTimeInRange(time, minTime, maxTime));
+        });
+
+        it("returns false if given time is greater than maxTime", () => {
+            const time = new Date(1995, 6, 30, 18, 41, 9, 50);
```

Comment:
Can we create a helper function to clarify that the day/month/year are ignored in all of these?

```tsx
function createTimeObject(hour: number, minute: number, second: number, millisecond: number) {
    const IGNORED_YEAR = 1995;
    const IGNORED_MONTH = 6;
    const IGNORED_DAY = 30;
    return new Date(IGNORED_YEAR, IGNORED_MONTH, IGNORED_DAY, hour, minute, second, millisecond);
}
```

Then, in each test, we can just do:

```tsx
const time = createTimeObject(13, 10, 50, 900);
```